src/app/main/component/events/components/event-editor/event-editor.component.spec.ts

```
@@ -1,0 +1,96 @@
+import { EventEditorComponent } from "./event-editor.component";
+
+describe('EventEditorComponent', () => {
+  let component: EventEditorComponent;
+  let fixture: ComponentFixture<EventEditorComponent>;
+  let loader: HarnessLoader;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [EventEditorComponent],
+      imports: [
+        EventsModule,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)